data/remnant missions.txt

```diff
@@ -576,7 +576,7 @@ mission "Remnant: Technology Available"
 	on offer
 		payment 2000000
 		set "license: Remnant"
-		log "The prefects have reached a consensus that you should be considered a valuable friend of the Remnant, and shall be given permission to purchase their ships and technology."
```

Comment:
iirc this line has been debated before. @Zitchas ?

Reply:
This line is being fixed in #4706.